bindings/cpp/include/AutoSystem.hxx

```diff
@@ -3,7 +3,7 @@
 
 // ---------------------------------------------------------------------------------------
 
-#include "portaudiocpp/System.hxx"
+#include "System.hxx"
```

Comment:
Whoops, I think this would break user applications. Do we need to put the headers back in `bindings/cpp/include/portaudiocpp` or and set the include path to `bindings/cpp/include` or is there a way we can keep the headers in `bindings/cpp/include` and keep the `#include`s in the C++ code referring to "portaudio/headername.hxx"?